anvil-python/pyproject.toml

```diff
@@ -26,6 +26,7 @@ dynamic = [
 ]
 dependencies = [
   "pydantic==1.10.14",
+  "python-openstackclient===6.6.0",
```

Comment:
@skatsaounis Will this need the requirements to be re-created, or will that happen automatically?

Reply:
It will not happen automatically. Since we are not using the openstack cli at all, could you please get rid of the part of sunbeam that we include and it is using it? If yes, we can safely remove this addition as well

Reply:
Removed, had to duplicate the LatestInChannel construct